### ISSUE TITLE
Set custom uid:gid for sftp virtual folder.

### ISSUE THREAD

**Issue by dronmaxman** [drakkan/sftpgo#1738]
### Describe the solution you'd like

I want to provide access to several web servers using a single sftpgo server. I configured a virtual folder on the sftpgo server and connected it via sftp to the web server from root.

The problem I have is.

The files on the web server have the rights www-data:www-data, and the virtual folder is connected from root, when I make changes, the rights change to root:root and accordingly the web server loses access to these files.

### Describe alternatives you've considered

Create additional settings in the virtual folder where you can specify the permissions from which the files will be stored.

### What are you using SFTPGo for?

Private user, home usecase (home backup/VPS)

**Comment by ravensorb** [comment]
Having the same issue -- this would be a great feature!

**Comment by NicoleCollard** [comment]
Yes, here the same a few sftp user want to upload in the same folder, this is located on a nas.

**Comment by Eliophot** [comment]
Hello, same issue with docker logic...
I think similar issue : https://github.com/drakkan/sftpgo/issues/1738
Thanks

**Comment by Rafq998** [comment]
Were you able to add a folder from a NAS, such as a Synology, to SFTPGO from a computer on your LAN?
I have SFTPGO on proxmox.
